Core/include/Acts/Seeding/SeedFilterConfig.hpp

```diff
@@ -37,11 +37,48 @@ struct SeedFilterConfig {
   // sort vectors vectors by curvature
   bool curvatureSortingInFilter = false;
 
+  // increment in seed weight if the number of compatible seeds is larger than
+  // numSeedIncrement, this is used in case of high occupancy scenarios if we
+  // want to increase the weight of the seed by seedWeightIncrement when the
+  // number of compatible seeds is higher than a certain value
+  float seedWeightIncrement = 0;
+  float numSeedIncrement = std::numeric_limits<float>::infinity();
+
+  // seedConfirmation enables seed confirmation cuts - keep seeds if they have
+  // specific values of impact parameter, z-origin and number of compatible
+  // seeds inside a pre-defined range that also depends on the region of the
+  // detector (i.e. forward or central region) defined by SeedConfirmationRange
+  bool seedConfirmation = false;
+  // contains parameters for central seed confirmation
+  SeedConfirmationRangeConfig centralSeedConfirmationRange;
+  // contains parameters for forward seed confirmation
+  SeedConfirmationRangeConfig forwardSeedConfirmationRange;
+  // minimum radius for bottom SP in seed confirmation
+  float seedConfMinBottomRadius = 60. * Acts::UnitConstants::mm;
+  // maximum zOrigin in seed confirmation
+  float seedConfMaxZOrigin = 150. * Acts::UnitConstants::mm;
+  // minimum impact parameter for seed confirmation
+  float minImpactSeedConf = 1. * Acts::UnitConstants::mm;
+
+  // maximum number of lower quality seeds in seed confirmation
+  float maxSeedsPerSpMConf = std::numeric_limits<float>::infinity();
```

Comment:
May be better to use int instead of floats here

Reply:
I'm not sure what happens when you assign float::infinity to an int. int doesn't have infinity, only std::numeric_limits<int>::max(). Assign that instead

Reply:
ok!